list.go

```diff
@@ -46,6 +46,10 @@ func runListCommand() {
 	}
 
 	feedback.PrintResult(listResult{Latest: manifest.Latest, Releases: manifest.Releases})
+	err = checkForUpdates()
+	if err != nil {
+		feedback.Warning("\n\nfailed to check for updates: " + err.Error())
+	}
```

Comment:
We cannot check for updates at every command without a timeout of 1 day, for instance, and we cannot print a warning if the user is not connected to the internet.

As I said in another comment, for now we should only do a check in the version command, otherwise we need to make this smarter and store in a cache file the last time we check for an update, and at least don't do a check more the 1 per day.